src/prpy/planning/base.py

```diff
@@ -298,6 +318,23 @@ def plan(self, method, args, kw_args):
                     with Timer() as timer:
                         output = planner_method(*args, **kw_args)
 
+                    if not is_sequence_deterministic:
+                        # TODO: It is overly conservative to set _ENDPOINT,
+                        # e.g. for PlanToConfiguration. Unfortunately, there is
+                        # no easy way to detect this special case.
+                        SetTrajectoryTags(output, {
+                            Tags.DETERMINISTIC_TRAJECTORY: False,
+                            Tags.DETERMINISTIC_ENDPOINT: False,
+                        }, append=True)
+
+                        if not self._allow_nondeterministic:
+                            logger.warning(
+                                'Tagging trajectory as non-deterministic because an'
+                                ' earlier planner in the Sequence threw a'
+                                ' non-deterministic PlanningError. Pass the'
+                                ' "allow_nondeterministic" to this Sequence'
+                                ' constructor if you intended this behavior.')
+
```

Comment:
It's an option to suppress the warning. It's technically valid to put a deterministic planner after a non-deterministic planner, but it's generally a bad idea because you may end up re-running the deterministic planner on the same problem again.